src/Controls/src/Core/Compatibility/Handlers/Shell/iOS/ShellNavBarAppearanceTracker.cs

```diff
@@ -115,6 +122,16 @@ void UpdateiOS15NavigationBarAppearance(UINavigationController controller, Shell
 			if (titleColor != null)
 				navigationBarAppearance.TitleTextAttributes = new UIStringAttributes() { ForegroundColor = titleColor.ToPlatform() };
 
+			// if the BackgroundColor is set to transparent, let's apply translucence as well
+			// since we cannot call this directly in .NET 8
+			if (appearance.BackgroundColor == Colors.Transparent)
+			{
+				// save the shadow image
+				_defaultNavBarShadowImage = navBar.ShadowImage;
+				navigationBarAppearance.ConfigureWithTransparentBackground();
```

Comment:
from the description of ConfigureWithTransparentBackground it seems like you'd only call that if there's no alpha value